core/src/main/scala/chisel3/Module.scala

```diff
@@ -259,6 +260,7 @@ package experimental {
   /** Abstract base class for Modules, an instantiable organizational unit for RTL.
     */
   // TODO: seal this?
+  @nowarn("msg=class Port") // delete when Port becomes private
```

Comment:
We should probably just promote this warning suppression to build.sbt `-Wconf` level, or go ahead and make the class package private and remove the deprecation.

Reply:
Do you want me to do it in this PR?